Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml

```diff
@@ -264,6 +313,19 @@
     - state: arnold
   - type: SliceableFood
     slice: FoodPizzaArnoldSlice
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 85
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 30
+        - ReagentId: Vitamin
+          Quantity: 5
+        - ReagentId: Iron
+          Quantity: 10
+        - ReagentId: Omnizine
+          Quantity: 30
```

Comment:
thats a lot of omnizine

Reply:
Not cookable right now + there's other reasons you wouldn't want to eat it with its special mechanics

Reply:
I added stuff for a lot of food that isn't actually able to be made right now but will be in the future

Reply:
sure